vsts/vsts.yaml

```diff
@@ -419,12 +421,15 @@ jobs:
           choco install -y squid
 
         displayName: "Install Squid"
+        condition: eq(variables.SHOULD_RUN, 'True')
 
       - powershell: ./vsts/start_tpm_windows.ps1
```

Comment:
nit: if this is the windows build, should we use backslashes?

Reply:
Nope, it works either way.